Open_Decision_Framework-community-v1.0.md

```diff
@@ -0,0 +1,328 @@
+
+Open decisions are made using open source principles
+====================================================
+
+**Open exchange**
+
+Whether you're developing software or trying to solve a business problem, open exchange begins when you share your "source code" with others. A free exchange of ideas is critical to creating an environment where people are allowed to learn and use existing information toward creating new ideas.
+
+**Participation**
+
+When we are free to collaborate, we create. We can solve problems that no one person may be able to solve on their own. And when we can implement open standards, we enable others to participate in the future.
+
+**Release early + often**
+
+Rapid prototypes can lead to rapid failures, and that leads to better solutions faster. When you're free to experiment, you can look at problems in new ways and look for answers in new places. You can learn by doing.
+
+**Meritocracy**
+
+In a meritocracy, good ideas can come from anywhere, and the best ideas win. Everyone has access to the same information. Successful work determines which projects rise and gather support and effort from the community.
+
+**Community**
+
+Communities are formed around a common purpose. They bring together diverse ideas and share work. Together, a global community can create beyond the capabilities of any one individual. It multiplies effort and shares the work. Together, we can do more.
+
+Adapted from: https://opensource.com/open-source-way
+
+How open source principles lead to better decisions 
+===================================================
+
+**Principles**
+
+  - Open exchange
+  - Participation
+  - Release early + often
+  - Meritocracy
+  - Community
+
+Lead to:
+
+**Practices**
+
+  - Transparency with internal customers and other stakeholders
+  - Customer involvement
+  - Gain feedback and adapt iterative changes
+  - Ideation with customers
+  - Build trust and respect via collaboration 
+
+Lead to:
+
+**Outcomes**
+
+  - Customer buy-in
+  - Stronger and faster adoption
+  - Best ideas win
+  - Fewer bugs, issues, and unanticipated impacts
+  - Higher associate engagement
+  - Decisions aligned to strategy and culture
+
+You can't please everyone. But when you make open decisions, people feel... 
+===========================================================================
+
+  - I understand why the decision was made and how it aligns to Red Hat's strategy, goals, and mission.
+  - There was visibility to the business requirements, research, and evaluation criteria.
+  - The decision-making process was inclusive and transparent.
+  - Although I wasn't the decision maker, I was able to contribute to the process.
+  - I may not agree with the decision, but it's obvious that the decision makers understand Red Hat's values and culture.
+  - I might be disappointed, but I wasn't surprised.
+  - My voice was heard and valued.
+
```

Comment:
Do we need a heading here to indicate that the phases are coming, or to "preview" the phases with a list before diving in?